Framework/Geometry/src/Objects/ShapeFactory.cpp

```diff
@@ -1307,6 +1302,55 @@ V3D ShapeFactory::parsePosition(Poco::XML::Element *pElem) {
   return retVal;
 }
 
+/** Create a hexahedral shape object
+@param xlb :: Left-back x point or hexahedron
+@param xlf :: Left-front x point of hexahedron
+@param xrf :: Right-front x point of hexahedron
+@param xrb :: Right-back x point of hexahedron
+@param ylb :: Left-back y point or hexahedron
+@param ylf :: Left-front y point of hexahedron
+@param yrf :: Right-front y point of hexahedron
+@param yrb :: Right-back y point of hexahedron
+
+@returns the newly created hexahedral shape object
+*/
+boost::shared_ptr<Object>
+ShapeFactory::createHexahedralShape(double xlb, double xlf, double xrf,
+                                    double xrb, double ylb, double ylf,
+                                    double yrf, double yrb) {
+  Hexahedron hex;
+  hex.lbb = V3D(xlb, ylb, 0);
+  hex.lbt = V3D(xlb, ylb, 0.001);
```

Comment:
Where does `1e-3` come from? Shouldn't we have this as a `static const` somewhere?